libs/application/core/src/types/ApplicationTemplate.ts

```diff
@@ -10,13 +10,15 @@ import {
 import { ApplicationTypes } from './ApplicationTypes'
 import { Schema, StaticText } from './Form'
 import { AnswerValidator } from '../validation/AnswerValidator'
+import { ApplicationFeatures } from './ApplicationFeatureFlags'
 
 export interface ApplicationTemplate<
   TContext extends ApplicationContext,
   TStateSchema extends ApplicationStateSchema<TEvents>,
   TEvents extends EventObject
 > {
   readonly readyForProduction?: boolean
```

Comment:
Yeah that was the plan, just wanted to see this on prod before going into a enforcing it on all applications... Just in case I might have missed something 😅

Reply:
Just so I have the feature flag implementation understanding correct.

We implement all feature flags for all applications, enable those that should be active and disable the ones that should and then remove the readyForProd flag.
Do we enforce the FeatureFlag on the applications templates so we dont run into the same issue as we did when readyForProd was introduced where people were forgetting to add the flag?  That does tho introduce a new problem where we want applications that dont have the configcat featureflag to be visible to all those applications are breaking the required type, how do we solve that?